testing/adios2/engine/bp/TestBPJoinedArray.cpp

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
Put this in staging-common so that we're ready to test this in BP5, SST, etc?

Reply:
Task for another PR. I want to push this out for Axel a.s.a.p